src/main/java/co/inventorsoft/academy/collections/model/Range.java

```diff
@@ -1,60 +1,138 @@
+//        System.out.println(a.add(0.2f));
+//        System.out.println(a.contains(5));
+//        a.test();
+    }
+
+    public Range(float a, float b) {
+        if(!(a==b)){
+            while (a <=b) {
+                mylist.add(a);
+                a = a + 0.1f;
+                a = a * 10;
+                a = Math.round(a);
+                a = a / 10;
+
+            }}
+    }
+
+    public Range(int a, int b) {
+        if (!(a==b)){
+            while (a <= b) {
+                mylist.add(a);
+                a = a + 1;
+            }}
+
+    }
+
+    public static Range of(int i, int i1) {
+        return new Range(i, i1);
+    }
+
+    public static Range of(float i, float i1) {
+        return new Range(i, i1);
+    }
+
+    public static Range of(char i, char i1) {
+        return new Range(i, i1);
+    }
+
+    public static Range<Character> of(char a, char d, Function<Character, Character> characterCharacterFunction) {
+        return null;
+    }
+
     public int size() {
-        return 0;
+        return mylist.size();
     }
 
     public boolean isEmpty() {
-        return false;
+        return mylist.isEmpty();
     }
 
+    @Override
     public boolean contains(Object o) {
-        return false;
+        return mylist.contains(o);
     }
 
-    public Iterator<T> iterator() {
-        return null;
+    @Override
+    public Iterator iterator() {
+        return mylist.iterator();
     }
 
+    @Override
     public Object[] toArray() {
         return new Object[0];
     }
 
-    public <T1> T1[] toArray(T1[] a) {
-        return null;
+    @Override
+    public boolean add(Object o) {
+        return mylist.add(o);
+    }
+
+    @Override
+    public boolean remove(Object o) {
+        return mylist.remove(o);
     }
 
-    public boolean add(T t) {
-        return false;
+    @Override
+    public boolean addAll(Collection c) {
+        return mylist.addAll(c);
     }
 
-    public boolean remove(Object o) {
-        return false;
+    @Override
+    public void clear() {
+        mylist.clear();
     }
 
-    public boolean containsAll(Collection<?> c) {
-        return false;
+    @Override
+    public boolean removeAll(Collection c) {
+        return mylist.removeAll(c);
     }
 
-    public boolean addAll(Collection<? extends T> c) {
-        return false;
+    @Override
+    public boolean retainAll(Collection c) {
+
+        return mylist.retainAll(c);
     }
 
-    public boolean retainAll(Collection<?> c) {
-        return false;
+    @Override
+    public boolean containsAll(Collection c) {
+
+        return mylist.containsAll(c);
     }
 
-    public boolean removeAll(Collection<?> c) {
-        return false;
+    @Override
+    public Object[] toArray(Object[] a) {
+        return new Object[0];
     }
 
-    public void clear() {
+    public boolean contains(float o) {
+        return mylist.contains(o);
+    }
+    public boolean contains(int o) {
```

Comment:
Use object types, no need to overload for primitive types

Reply:
corrected